src/gentypes.jl

```diff
@@ -109,7 +111,7 @@ function _pkgtype_import(input::Expr)
     @assert input.args[1].head == :(.) "Improper @rosimport input"
     p = input.args[1].args[1]
     @assert isa(p, Symbol) "Package name ($(string(p))) not a symbol"
-    m_or_s = input.args[1].args[2].args[1]
+    m_or_s = _get_quote_value(input.args[1].args[2])
```

Comment:
Comes from this [`NEWS.md`](https://github.com/JuliaLang/julia/blob/master/NEWS.md) item:

QuoteNodes are also consistently used for quoted symbols instead of the :quote expression head (though :quote Exprs are still used for quoted expressions) ([#23885]).

Reply:
I had a vague suspicion that all these direct `.args` calls and indexing were going to come back and be an issue one day...